Testscripts/Linux/SRIOV-VerifyVF-Connection.sh

```diff
@@ -25,6 +25,51 @@ fi
     exit 0
 }
 
+function version_lt() { test "$(echo "$@" | tr " " "\n" | sort -rV | head -n 1)" != "$1"; }
+
+MIN_VERSION_SUPPORTED="6.7"
+
+if [ $DISTRO_NAME == "centos" ] || [ $DISTRO_NAME == "rhel" ] || [ $DISTRO_NAME == "oracle" ]; then
+    if version_lt $DISTRO_VERSION $MIN_VERSION_SUPPORTED ; then
+        LogErr "SRIOV is not supported for Distro's below $MIN_VERSION_SUPPORTED, Test skipped!"
+        SetTestStateSkipped
+        exit 0
+    fi
+fi
+
+# Check if Extra NICs have ips
```

Comment:
This logic is not so clear. This script is called in single VF as well as Max VF. In case of max VF, this block is not exercised.

Reply:
Added a check if [[ "$NIC_COUNT" -gt 1 ]];then 
in case of single VF, NIC_COUNT=1 , this block will not get executed because we are always getting IP
in case of Max VF, NIC_COUNT >1 , this block will get executed